docs/tutorials/t2hahn_characterization.ipynb

```diff
@@ -0,0 +1,507 @@
+      "text/plain": [
+       "<Figure size 576x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "dt_factor = apply_prefix(1, unit)\n",
```

Comment:
You're not using `dt_factor`, remove this line.

Reply:
deleted this variable.